Project EFT/Views/Shared/AddProblemPage.cshtml

```diff
@@ -0,0 +1,33 @@
+﻿@using Microsoft.AspNetCore.Http;
+@inject Microsoft.AspNetCore.Http.IHttpContextAccessor HttpContextAccessor
+@using Project_EFT.Data_Classes;
+@{ISession session = HttpContextAccessor.HttpContext.Session; }
+
+@if (session.ContainsKey("adminInfo"))
+{
+    <h1>You can create a new problem here!</h1>
+    <!-- submission form-->
+    <form action="~/Problem/AddProblem" method="post">
+        Enter Problem Title: <br />
+        <input type="text" id="title" name="title" /><br />
+        Enter Problem Question: <br />
+        <textarea name="question" id="question" cols="50" rows="4"> </textarea> <br />
+        Enter Problem Answer: <br />
+        <textarea name="answer" id="answer" cols="50" rows="4"> </textarea> <br />
+        <input type="submit" value="Submit your new problem!" />
+        <div></div>
+    </form>
+
+    @if (ViewData["message"] == null)
+    {
+
+    }
+    else
+    {
+        <div>@ViewData["message"]</div>
+    }
```

Comment:
if the value at that key is null, the div is empty -> no logic needed

Reply:
Updated this, and I used this same technique in a different place as well.  Fixed both.